doc/languages-frameworks/python.section.md

```diff
@@ -734,6 +734,24 @@ work in any of the formats supported by `buildPythonPackage` currently,
 with the exception of `other` (see `format` in
 [`buildPythonPackage` parameters](#buildpythonpackage-parameters) for more details).
 
+### Using unittestCheckHook {#using-unittestcheckhook}
+
+`unittestCheckHook` is a hook which will substitute the setuptools `test` command for a `checkPhase` which runs `python -m unittest discover`.
+
+```
+  checkInputs = [ unittestCheckHook ];
+
+  unittestFlags = [ "-s" "tests" "-v" ];
+```
+
+translates to:
```

Comment:
This seems a bit strange. The hook implements a phase, and that phase runs a pre and post hook. Also, in the above you pass in flags but in the bottom they're missing.

Reply:
How's this?

---

`unittestCheckHook` is a hook which will substitute the setuptools `test` command for a `checkPhase` which runs `python -m unittest discover`:

```nix
  checkInputs = [ unittestCheckHook ];

  unittestFlags = [ "-s" "tests" "-v" ];
```

Reply:
Sounds good to me.

Reply:
Done.